app/controllers/v0/sessions_controller.rb

```diff
@@ -256,7 +256,7 @@ def log_persisted_session_and_warnings
       # We want to log when SSNs do not match between MVI and SAML Identity. And might take future
       # action if this appears to be happening frequently.
       if current_user.ssn_mismatch?
-        additional_context = StringHelpers.heuristics(current_user.identity.ssn, current_user.va_profile.ssn)
+        additional_context = StringHelpers.heuristics(current_user.identity.ssn, current_user.mpi.profile.ssn)
```

Comment:
I feel like there are too many steps to get to SSN, could we do something like current_user.mpi_profile.ssn?

Reply:
Yeah, I actually don't like this that much anyway. I might remove `additional_context` altogether, and just show this log with the `user_uuid` is there's an SSN mismatch